src/engine/audio/Sound.cpp

```diff
@@ -206,6 +205,16 @@ namespace Audio {
         return currentGain;
     }
 
+    void Sound::SetVolumeModifier(const Cvar::Range<Cvar::Cvar<float>>& volumeModifier)
```

Comment:
why use a shadowing name? I find this an annoying thing and always have a doubt on scope. This will also trigger warnings.

Reply:
Done.